frontend/src/components/phones/dashboard/PhoneDashboard.module.scss

```
@@ -0,0 +1,264 @@
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "../../../styles/tokens.scss";
+
+.dashboard-card {
+  box-shadow: $box-shadow-sm;
+  background-color: $color-white;
+  border-radius: $border-radius-md;
+  display: flex;
+  flex-direction: column;
+  padding: $spacing-lg 0;
+
+  //   & > * {
+  //     padding: 0 $spacing-xl;
+  //   }
+
+  .header-phone-number {
+    @include text-body-xl;
+    @include font-firefox;
+    display: flex;
+    padding: 0 $spacing-xl;
+  }
+
+  .phone-statistics-container {
+    margin: $spacing-md $spacing-lg 0 $spacing-lg;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-radius: $border-radius-lg;
+    border: 2px solid $color-light-gray-20;
```

suggestion (non-blocking): 
```suggestion
    border: $border-radius-xs solid $color-light-gray-20;
```